.pipelines/1es-migration/azure-pipelines.yml

```diff
@@ -92,151 +80,242 @@ extends:
 
     stages:
 
-    - stage: BuildAndTest
-      displayName: 'Build and Test ${{ parameters.extensionName }}'
+    # ============================== Build (MAIN+TEST) ==============================
+    - stage: Build
+      displayName: 'Build ${{ parameters.extensionName }} (MAIN + TEST)'
       jobs:
-      - job: BuildAndTestJob
-        displayName: 'Build Extension'
+      - job: BuildJob
+        displayName: 'Build and Package MAIN, then TEST'
         templateContext:
           outputs:
           - output: pipelineArtifact
-            displayName: 'Publish Build Artifacts (Unsigned)'
-            targetPath: '$(System.ArtifactsDirectory)'
+            displayName: 'Publish MAIN Build Artifacts (Unsigned)'
+            targetPath: '$(System.ArtifactsDirectory)/unsigned'
             artifactName: 'vsix-unsigned'
+          - output: pipelineArtifact
+            displayName: 'Publish TEST Build Artifacts (Unsigned)'
+            targetPath: '$(System.ArtifactsDirectory)/test-unsigned'
+            artifactName: 'vsix-test-unsigned'
         steps:
         - checkout: self
-          displayName: 'Checkout Extensions Repository'
+          displayName: 'Checkout'
           clean: true
           fetchTags: false
 
         - task: NodeTool@0
           displayName: 'Install Node.js'
-          inputs:
-            versionSpec: '10.x'
+          inputs: { versionSpec: '10.x' }
 
         - task: Npm@1
           displayName: 'Install Dependencies'
-          inputs:
-            command: 'install'
-            verbose: false
+          inputs: { command: 'install', verbose: false }
 
         - task: Npm@1
           displayName: 'Install TFX CLI'
-          inputs:
-            command: 'custom'
-            customCommand: 'install -g tfx-cli' 
+          inputs: { command: 'custom', customCommand: 'install -g tfx-cli' }
 
         - task: Npm@1
           displayName: 'Install Gulp CLI'
-          inputs:
-            command: 'custom'
-            customCommand: 'install -g gulp-cli'
+          inputs: { command: 'custom', customCommand: 'install -g gulp-cli' }
 
         - task: PowerShell@2
-          displayName: 'Build Extensions with Gulp'
-          inputs:
-            targetType: 'inline'
-            script: 'gulp build'
-            workingDirectory: '$(Build.SourcesDirectory)'
+          displayName: 'Build MAIN - gulp build'
+          inputs: { targetType: inline, script: 'gulp build', workingDirectory: '$(Build.SourcesDirectory)' }
```

Comment:
Any reason why you switched to this syntax? Yaml files usually follow the approach below and it's more readable or easier to extend if we need to add new properties. I commented this line, but it applies to the whole file.

```
- task: PowerShell@2
          displayName: 'Build Extensions with Gulp'
          inputs:
            targetType: 'inline'
            script: 'gulp build'
            workingDirectory: '$(Build.SourcesDirectory)'
```

Reply:
You're right, that standard yaml is more readable. Fixed it👌